packages/shared/src/dashboards/breakdown-card/breakdown-body.tsx

```diff
@@ -59,16 +60,18 @@ export const BreakdownCardBody: React.FC<BreakdownBodyProps> = ({
     <Grid>
       <GridItem span={4}>
         <TotalCapacityBody
-          capacity={humanize(capacityUsed).string}
-          suffix={t('used')}
+          prefix={isPersistentInternal && t('Total requests: ')}
+          capacity={humanize(metricTotal).string}
+          suffix={isPersistentInternal ? undefined : t('used')}
+          styleCapacityAsBold={isPersistentInternal}
         />
       </GridItem>
       <GridItem span={4} />
       <GridItem span={4}>
         {capacityAvailable && (
           <TotalCapacityBody
             capacity={humanize(capacityAvailable).string}
-            suffix={t('available')}
+            suffix={isPersistentInternal ? undefined : t('available')}
```

Comment:
nits: type of suffix is string...
```suggestion
            suffix={isPersistentInternal ? '' : t('available')}
```

Reply:
or don't pass the prop at all, if it is optional prop...

Reply:
done